### DISCUSSION TITLE
Error vaultwarden::api::admin][ERROR] Invalid admin token.

### DISCUSSION THREAD

**Discussion by mikey6283** [dani-garcia/vaultwarden#2714] [Q&A]
FInally thought i had  Vault Warden connected to cloudflare. It was ok for two days then i noticed that cloudflare had errors so i restarted and it was fine but now i unable to log into admin. created a new key  still unable to login

| This is an *unofficial* Bitwarden implementation, DO NOT use the |
| official channels to report bugs/features, regardless of client. |
| Send usage/configuration questions or feature requests to: |
| https://vaultwarden.discourse.group/ |
| Report suspected bugs/issues in the software itself at: |
| https://github.com/dani-garcia/vaultwarden/issues/new |

2022-08-25 15:00:04.910][vaultwarden::api::admin][ERROR] Invalid admin token. IP: 192.168.0.16

[2022-08-25 15:00:04.910][response][INFO] (post_admin_login) POST /admin/ => 303 See Other
[2022-08-25 15:00:04.914][request][INFO] GET /admin
[2022-08-25 15:00:04.914][_][WARN] Request guard `AdminToken` is forwarding.
[2022-08-25 15:00:04.914][response][INFO] (admin_login) GET /admin/ [2] => 200 OK

 FInally thought i had  Vault Warden connected to cloudflare. It was ok for two days then i noticed that cloudflare had errors so i restarted and it was fine but now i unable to log into admin. created a new key  still unable to login

 I am not sure is these errors have anything to do with it but hopefully it may help.

I hope you can advice?

**Comment by BlackDex** [comment]
There is a similar case regarding this:
https://github.com/dani-garcia/vaultwarden/discussions/2448

For some reason, sometimes, cloudflare has some issues with the 303 response.
That is not something we can fix on our side.

**Comment by FatGrizzly** [comment]
> There is a similar case regarding this: #2448
> 
> For some reason, sometimes, cloudflare has some issues with the 303 response. That is not something we can fix on our side.

I am currently running vaultwarden behind cloudflare for more than 5 months, have had no issues logging into /admin.
